### PR TITLE
Add support for MIT license

### PR DESCRIPTION
**Pull request by natansil**
## What is the goal of this PR?

Add support for MIT license in pom.xml template

## What are the changes implemented in this PR?

added MIT as an option for `assemble_maven`  license field

### CODE REVIEW

**Comment by natansil** [comment]
FYI @vmax

**Comment by natansil** [comment]
@sorsaffari may you can take a look?

File: maven/templates/rules.bzl [hunk 1]
```diff
@@ -337,7 +340,7 @@ assemble_maven = rule(
             doc = 'Project URL to fill into pom.xml'
         ),
         "license": attr.string(
-            values=["apache"],
+            values=["apache", "MIT"],
```
Comment (vmax) [inline maven/templates/rules.bzl:343]: @natansil could we use `mit` here (lowercase)? we're trying to achieve consistent naming everywhere

**Comment by vmax** [comment]
@natansil thank you for your contribution! small change requested

**Comment by natansil** [comment]
Thank you for the review @vmax!

Changed to lowercase.
I've also added license text substition according to choice of apache and mit.

**Comment by natansil** [comment]
@sorsaffari @vmax can one of you please review this after my changes.

**Comment by natansil** [comment]
@sorsaffari @vmax friendly ping

**Pull request merged**